lib/tasks/notify.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :notify do
+  desc "Send an email notification"
+  task :send_email, [:email_address] => :environment do |_, args|
+    raise "Missing email address" unless args.email_address
+
+    headers = {
```

Comment:
minor, but I don't think these are headers and more parameters.

Reply:
I've used `headers` here as it's the terminology that's used in Action Mailer https://guides.rubyonrails.org/action_mailer_basics.html#complete-list-of-action-mailer-methods

Reply:
Ah yes I see what you mean. I don't think that is referring to the params for mail though - looks like that it's a different hash in a mailer for the email headers. The source code describes it better: https://github.com/rails/rails/blob/1cdc6c6576afe9a9fa67a6b313080a7dc62188d9/actionmailer/lib/action_mailer/base.rb#L50-L55